table_bindings.go

```diff
@@ -169,8 +169,16 @@ func (t *TableMap) bindInsert(elem reflect.Value) (bindInstance, error) {
 }
 
 func (t *TableMap) bindUpdate(elem reflect.Value, colFilter ColumnFilter) (bindInstance, error) {
+
 	if colFilter == nil {
+		if t.colFilter != nil {
+			t.updatePlan = bindPlan{}
+		}
+		t.colFilter = nil
 		colFilter = acceptAllFilter
+	} else {
+		t.updatePlan = bindPlan{} // a new bindPlan everytime; since we cannot compare colFilter funcs
+		t.colFilter = colFilter
```

Comment:
I'm a firm believer in avoiding `else` at nearly all costs.  Branches are really harsh on the mental stack when reading code, and `else` statements are the worst (especially in go, where they can carry extra context).  Can we try something more like:
```go
if t.colFilter != nil || colFilter != nil {
    t.updatePlan = bindPlan{}
}
if colFilter == nil {
    colFilter = acceptAllFilter
}
t.colFilter = colFilter
```